ytfzf

```diff
@@ -318,21 +318,19 @@ get_search_query () {
 	else
 		if [ -z "$search_query" ]; then
 			if [ $is_ext_menu -eq 1 ]; then 
-				search_query="$(printf "" | $YTFZF_EXTMENU)"
+				search_query="$(eval $YTFZF_EXTMENU)"
```

Comment:
This is needed cause if you do just 
```
ytfzf -D
```
Then you get a search prompt. Dmenu and rofi needs some stdin.

Reply:
if you do just `dmenu ` you wont get a prompt. You need some stdin.